dali/base/dautils.cpp

```diff
@@ -65,6 +65,29 @@ IPropertyTree * getDropZonePlane(const char * name)
     return getGlobalConfigSP()->getPropTree(xpath);
 }
 
+StringBuffer & makeDropZoneScope(StringBuffer & scope, bool isPlane, const char *dropZone, const char *path)
+{
+    scope.append(isPlane ? PLANE_SCOPE : EXTERNAL_SCOPE).append("::").append(dropZone);
+    const char* s = path;
+    while (*s)
+    {
+        if (isPathSepChar(*s))
+        {
+            s++;
+            if (*s)//Skip Trailing Path Sep Char
+                scope.append("::");
+        }
+        else
+        {
+            if ((*s=='^') || isupper(*s))
+                scope.append('^'); //Do we need to convert the upper cases here?
```

Comment:
not sure what this comment means? you do need to convert, but you are on next line

Reply:
The code does the conversion. I just want to double check (in case I should not convert). I will remove the comment.